src/component/handlers/edit/editOnBlur.js

```diff
@@ -28,8 +28,12 @@ function editOnBlur(editor: DraftEditor, e: SyntheticEvent): void {
   // issue to be certain, checking whether the active element is `body`
   // to force it when blurring occurs within the window (as opposed to
   // clicking to another tab or window).
-  if (isWebKit && getActiveElement() === document.body) {
-    global.getSelection().removeAllRanges();
+  var doc = e.target && e.target.ownerDocument instanceof Document ?
```

Comment:
What's the reason for this check? I'd expect e.target.ownerDocument would always be a document. Also, you need `e.target.ownerDocument.nodeType === 9` or else this won't work in multiple realms – exactly the case this is trying to support.